src/test/kotlin/study/Person.kt

```diff
@@ -1,24 +1,28 @@
 package study
 
-class Person {
-    lateinit var name: String
-    var company: String? = null
-    lateinit var skills: Skills
-    lateinit var languages: Languages
+data class Person(
+    val name: String = "",
+    val company: String = "",
+    val skills: Skills = Skills(),
+    val languages: Languages = Languages()
+)
 
-    fun name(name: String) {
-        this.name = name
-    }
+class PersonBuilder {
+    private var name: String = ""
+    private var company: String = ""
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private lateinit var name: String
    private var company: String? = null
```